product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/2.1-HTTP-to-HTTP-protocol-translation/2.1.1-http-to-http-passthrough/src/test/java/org/wso2/carbon/esb/scenario/test/HttpMessageWithoutAlteringDownstreamTest.java

```diff
@@ -38,18 +38,91 @@
  * downstream without altering the content
  */
 public class HttpMessageWithoutAlteringDownstreamTest extends ScenarioTestBase{
-    private final String carFileName = "2_1_1_7_httpToHttpProtocolTranslationCompositeApplication_1.0.0";
-    private String proxyServiceUrl;
+    private final String carFileName = "2_1_1_HttpMessageWithoutAlteringDownstreamTestCompositeApplication_1.0.0";
 
     @BeforeClass(alwaysRun = true)
     public void init() throws Exception {
         super.init();
-        proxyServiceUrl = getProxyServiceURLHttp("2_1_1_7_httpToHttpProtocolTranslationProxy");
         deployCarbonApplication(carFileName);
     }
 
-    @Test(description = "2.1.1.7 Testcase", enabled = true, dataProvider = "2.1.1.7")
+    @Test(description = "2.1.1.1", enabled = true, dataProvider = "2.1.1.1")
```

Comment:
Let's add a description in the following format
description = <ID> <Description>